src/main/java/org/jabref/gui/preferences/customization/CustomizationTabViewModel.java

```diff
@@ -23,19 +47,58 @@ public CustomizationTabViewModel(DialogService dialogService, PreferencesService
         this.dialogService = dialogService;
         this.preferencesService = preferencesService;
         this.initialDOIPreferences = preferencesService.getDOIPreferences();
+        initApiKeyNameUrl();
+    }
+
+    private void initApiKeyNameUrl() {
+        // Springer query using the parameter 'q=doi:10.1007/s11276-008-0131-4s=1' will respond faster
+        API_KEY_NAME_URL.put("Springer", "https://api.springernature.com/meta/v1/json?q=doi:10.1007/s11276-008-0131-4s=1&p=1&api_key=");
+        API_KEY_NAME_URL.put("IEEEXplore", "https://ieeexploreapi.ieee.org/api/v1/search/articles?max_records=0&apikey=");
+    }
+
+    /**
+     * Gets a list of Fetcher names which in the custom Tab
+     *
+     * @return ArrayList containing Fetcher names
+     */
+    public ArrayList<String> getCustomApiKeyFetchers() {
+        return new ArrayList<>(API_KEY_NAME_URL.keySet());
     }
 
+    /**
+     * Reads user settings when the dialog is opened.
+     */
```

Comment:
This comment is not specific for that tab, so general comments should go in the abstract class or interface, but not in this class

Reply:
deleted.